test/integration/test-video-players-helper.js

```diff
@@ -111,6 +117,168 @@ export function runVideoPlayerIntegrationTests(
     afterEach(cleanUp);
   });
 
+  describe.configure().retryOnSaucelabs().run('Video Docking', function() {
+    this.timeout(TIMEOUT);
+
+    describe('General Behavior', () => {
+      it('should have class when attribute is set (autoplay)', function() {
+        return getVideoPlayer(
+            {
+              outsideView: false,
+              autoplay: true,
+              dock: true,
+            }
+        ).then(r => {
+          return poll('checking class list', () => {
+            return !!r.video.classList.contains('i-amphtml-dockable-video');
```

Comment:
nit: no need for !! here and other places in this file. `contains` already returns boolean